.github/scripts/docs-prs-reviews.go

```diff
@@ -0,0 +1,356 @@
+			} `json:"pullRequest"`
+		} `json:"repository"`
+	} `json:"data"`
+}
+
+type review struct {
+	State     string    `json:"state"`
+	UpdatedAt time.Time `json:"updatedAt"`
+	Author    struct {
+		Login string `json:"login"`
+	} `json:"author"`
+}
+
+type gqlDocsPrsTeamMembers struct {
+	Data struct {
+		Organization struct {
+			Team struct {
+				Members struct {
+					Edges []struct {
+						Node struct {
+							Login string `json:"login"`
+						} `json:"node"`
+					} `json:"edges"`
+					PageInfo pageInfo `json:"pageInfo"`
+				} `json:"members"`
+			} `json:"team"`
+		} `json:"organization"`
+	} `json:"data"`
+}
+
+// parameters stores the GitHub API token, a dry run flag to output the issues it would create, and the
+// start and end times of the search.
+type parameters struct {
+	Token    string // GitHub API token
+	PrNumber int
+	PrAuthor string
+	IsDraft  bool
+}
+
+// pageInfo contains pagination information for querying the GraphQL API.
+type pageInfo struct {
+	HasNextPage bool   `json:"hasNextPage"`
+	EndCursor   string `json:"endCursor"`
+}
+
+type prReviewParse struct {
+	Approved  bool
+	UpdatedAt time.Time
+}
+
+const (
+	docsOrganization = "cockroachdb"
+	docsRepo         = "docs"
+	docsPrsTeam      = "docs-prs"
+)
+
+func main() {
+	params, err := defaultEnvParameters()
+	if err != nil {
+		log.Fatalf("error: %v\n", err)
+	}
+	docsPrsMembers, err := searchDocsPrsTeamMembers(params.Token)
+	if err != nil {
+		log.Fatalf("error: %v\n", err)
+	}
+	_, authorIsDocsPrsMember := docsPrsMembers[params.PrAuthor]
+	if authorIsDocsPrsMember {
+		fmt.Printf(`User "%s" is a member of docs-prs, so no docs-prs approval required.`, params.PrAuthor)
+		os.Exit(0)
+	}
+	fmt.Printf(`User "%s" is not a member of docs-prs. Checking for docs-prs team approval...\n`, params.PrAuthor)
+
+	prReviews, err := searchDocsPrReviewers(params.Token, params.PrNumber)
+	if err != nil {
+		log.Fatalf("error: %v\n", err)
+	}
+
+	isReviewed := parseDocsPrReviewers(prReviews, docsPrsMembers)
+	if isReviewed {
+		fmt.Println("This PR has been approved by a member of docs-prs. You are ready to merge!")
+		os.Exit(0)
+	}
+	log.Fatalln("error: You need approval from a member of docs-prs to merge this PR. Please add the group or at least one member of the group as a reviewer to this PR.")
+
+}
+
+func defaultEnvParameters() (parameters, error) {
+	const (
+		githubTokenEnv = "GITHUB_TOKEN"
+		prNumberEnv    = "PR_NUMBER"
+		prAuthorEnv    = "PR_AUTHOR"
+	)
+	prNumber, err := strconv.Atoi(maybeEnv(prNumberEnv, ""))
+	if err != nil {
+		return parameters{}, fmt.Errorf("error converting PR number %s, %w", prNumberEnv, err)
+	}
+	return parameters{
+		Token:    maybeEnv(githubTokenEnv, ""),
+		PrNumber: prNumber,
+		PrAuthor: maybeEnv(prAuthorEnv, ""),
+	}, nil
+}
+
+func maybeEnv(envKey, defaultValue string) string {
+	v := os.Getenv(envKey)
+	if v == "" {
+		return defaultValue
+	}
+	return v
+}
+
+func stringInSlice(a string, list []string) bool {
```

Comment:
This is unused now. Kill it with fire :D

Reply:
Done!